Assets/Scripts/GameManager/TimeManager.cs

```diff
@@ -0,0 +1,20 @@
+﻿using UnityEngine;
+
+public class TimeManager : MonoBehaviour
+{
+    private static bool isGamePaused = false;
+
+    public static bool IsGamePaused => isGamePaused;
+
+    public static void PauseGame()
+    {
+        isGamePaused = true;
+        Time.timeScale = 0f;
+    }
+
+    public static void UnpauseGame()
+    {
+        isGamePaused = false;
+        Time.timeScale = 1f;
```

Comment:
I'm not sure what this value should actually be @croconut but it seems to function correctly. I'm guessing this would have more of an impact when we do more time-based functionality.